### PR TITLE
Update "set approval" modal

### PR DESCRIPTION
**Pull request by epwinchell**
jira: https://projects.engineering.redhat.com/browse/SSP-1539

This PR set the modal title to the generic "Set Approval" and moves the full portfolio name into the body of the modal (just like "Share Portfolio".)  This allows long portfolio names to wrap properly.

Before
Screen Shot 2020-05-28 at 4 24 31 PM

After
Screen Shot 2020-06-01 at 10 28 04 AM

### CODE REVIEW

**Comment by epwinchell** [comment]
@sbuenafe-rh Any thoughts on the text here? It feels a little repetitive. Thanks.

Screen Shot 2020-05-28 at 4 27 01 PM

**Comment by sbuenafe-rh** [comment]
@epwinchell I would nix the dropdown field label **Select approval process**. I believe, the dropdown select would have a placeholder text that says this or something like Select a process.

For the body text, I would make sure it's using the PF body text and bold the resource (benny-test). I also would reword this to:
Select approval processes for **benny-test**

File: src/smart-components/common/edit-approval-workflow.js [hunk 1]
```diff
@@ -85,24 +92,34 @@ const EditApprovalWorkflow = ({
...
-        />
-      ) : (
-        <WorkflowLoader />
+      {isFetching && <WorkflowLoader />}
+      {!isFetching && (
+        <Stack gutter="md">
+          <StackItem>
+            <TextContent>
+              <Text component={TextVariants.h6}>
+                {`Set approval process for ${objectName(query[querySelector])}`}
```
Comment (Hyperkid123) [inline src/smart-components/common/edit-approval-workflow.js:106]: The template strings are not necessary in JSX outside of props definition
```jsx
<Text>
  Set approval process for {objectName(query[querySelector])}
</Text>
```

File: src/smart-components/common/edit-approval-workflow.js [hunk 2]
```diff
@@ -85,24 +92,34 @@ const EditApprovalWorkflow = ({
 
   return (
     <Modal
-      title={`Set approval process for ${objectName(query[querySelector])}`}
+      title={`Set approval process`}
```
Comment (Hyperkid123) [inline src/smart-components/common/edit-approval-workflow.js:95]: `title="Set approval process"` is good enough when you don't need variables.

File: src/test/smart-components/common/edit-approval-workflow.test.js [hunk 3]
```diff
@@ -143,11 +142,7 @@ describe('<EditApprovalWorkflow />', () => {
     });
 
     wrapper.update();
-    const modal = wrapper.find(Modal);
```
Comment (Hyperkid123) [inline src/test/smart-components/common/edit-approval-workflow.test.js:146]: Modal should be still in the virtual DOM
